secadvisor/pkg/transform_test.go

```diff
@@ -168,25 +156,25 @@ func Test_Transform_basic_flow(t *testing.T) {
 	f := getFlow()
 	transformer.extendGremlin.populateExtendGremlin(f, transformer)
 	secAdvFlow := transformer.Transform(f).(*SecurityAdvisorFlow)
-	assertEqual(t, version, secAdvFlow.Version)
-	assertEqualInt64(t, 0, secAdvFlow.UpdateCount)
-	assertEqual(t, "STARTED", secAdvFlow.Status)
-	assertEqualInt64(t, 1546338030000, secAdvFlow.Start)
-	assertEqualInt64(t, 1546338030000, secAdvFlow.Last)
-	assertEqual(t, "fake_node_type", secAdvFlow.NodeType)
+	core.AssertEqual(t, version, secAdvFlow.Version)
```

Comment:
Where is the definition of `core.AssertEqual`? I see no changes to files under `core`.

Reply:
added